docs/reference/user-variables.rst

```diff
@@ -208,6 +208,58 @@ and have some usage peculiarities:
   for build. Combining with previous peculiarity it's expected that much
   more disk space will be used than usually.
 
+.. _hunter download server:
+
+HUNTER_DOWNLOAD_SERVER
+======================
+
+When using Hunter without internet access in an internal LAN a download server
```

Comment:
> without internet access

I don't think it's an only case. Can be useful even with normal internet access, e.g. for speeding up download if regular server is slow.